css/styles.css

```diff
@@ -145,12 +158,14 @@
 .ab-column > ul > li label {
     cursor: default;
     display: inline-block;
-    width: 286px;
+    line-height: 24px;
+    max-width: 286px;
```

Comment:
In smaller viewports the max-width version seemed to be more robust and as you already mentioned is not harming any other behavior.